NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.ts

```diff
@@ -152,7 +152,7 @@ export class MyInterviewComponent implements OnInit {
         ).getTime();
         this.interviewService.updateInterview(interview).subscribe(res => {
           this.updateSuccess=true;
-    //      location.reload();
+          location.reload();
```

Comment:
We should listen to the observable to update the interviews. Reloading the page in a single page will cause us to lose performance by dropping the state of the components.